src/ebuckets.c

```diff
@@ -1398,7 +1398,7 @@ int ebRemove(ebuckets *eb, EbucketsType *type, eItem item) {
  * @param item - The eItem to be added to the ebucket.
  * @param expireTime - The expiration time of the item.
  *
- * @return 1 if the item was successfully added; Otherwise, return 0 on failure.
+ * @return 0 if the item was successfully added; Otherwise, return 1 on failure.
```

Comment:
These are the returned values. I only corrected the comment so that it'll correctly reflect what is being returned. Suggestions on the values themselves should probably be addressed by @moticless.

Reply:
I agree. it should return 1 on success. @ronen-kalish , if you already touching there, maybe you can fix it. Thanks.

Reply:
@moticless, sure. Will do.

Reply:
Changed that to return C_OK and C_ERR. Note that this requires including server.h, which also requires removing other include statements to avoid build errors.

Reply:
OK, changed it back to 0 on success and -1 on error (which are identical to C_OK and C_ERR)

Reply:
i'm fine with either ways, but i hope that it's better for ebuckets to have the same way internally.